frontend/src/metabase/dashboard/utils.ts

```diff
@@ -207,7 +206,7 @@ export function getDashcardResultsError(datasets: Dataset[]) {
   const isAccessRestricted = datasets.some(
     s =>
       s.error_type === SERVER_ERROR_TYPES.missingPermissions ||
-      s.error?.status === 403,
+      (typeof s.error === "object" && s.error?.status === 403),
```

Comment:
```suggestion
      (typeof s.error === "object" && s.error.status === 403),
```

Reply:
No because typeof x === “object” for null